src/wx/panel.cpp

```diff
@@ -2004,7 +2008,7 @@ BasicDrawingPanel::BasicDrawingPanel(wxWindow* parent, int _width, int _height)
     // 16 or 32, though
     if (gopts.filter == FF_NONE && gopts.ifb == IFB_NONE)
         // changing from 32 to 24 does not require regenerating color tables
-        systemColorDepth = 24;
+        systemColorDepth = 32;
```

Comment:
@rkitover @retro-wertz When you use `24`, the code for the Simple output seems to not allocated enough memory for everything. Then, we wait for whenever there is a wrong memory access when loading a state (can happen after {1,3,11,...} try(ies)). Depending where it lands, we get a different error message.